source/administration/changelog.md

```diff
@@ -4,10 +4,13 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
-## Release v3.8.1
+## Release v3.8.2
 
 ### Notes on Patch Release
 
+ - **v3.8.2, released 2017-04-21**
+   - Changed on the client to use `window.location.origin` over SiteURL, fixing WebSocket connection issues with Mattermost 3.8 upgrade.
```

Comment:
Should that be "Changed the client to use window.location.origin instead of siteURL, ..." ?